src/main/java/org/springframework/data/tarantool/exceptions/TarantoolMetadataMissingException.java

```diff
@@ -0,0 +1,18 @@
+package org.springframework.data.tarantool.exceptions;
+
+import io.tarantool.driver.exceptions.TarantoolException;
+
+/**
+ * When we expect only a flatted tuple,
+ * but we cannot convert the values into object fields because missing metadata.
+ *
+ * @author Artyom Dubinin
+ */
+public class TarantoolMetadataMissingException extends TarantoolException {
+    private static final String prefix = "Missing metadata for space ";
+    private static final String postfix = " check name in Entity class";
+
+    public TarantoolMetadataMissingException(String spaceName) {
+        super(prefix + spaceName + postfix);
```

Comment:
Use `String#format`, it is more readable

Reply:
Fixed